enb/EvolvedNodeB.m

```diff
@@ -44,8 +45,13 @@
 	
 	methods
 		% Constructor
-		function obj = EvolvedNodeB(Config, BsClass, cellId, Logger)
+		function obj = EvolvedNodeB(Config, Logger, BsClass, position, cellId, antennaBearing, siteId, macroCellId)
```

Comment:
Long function fingerprint. A solution has been discussed involving the use of the already created (and stored) network layout inside the Config object.

Reply:
Resolved in latest commits with a site and cell configuration structure that is generated in the layout (for the sites) and in the site constructor (for the cells)